app/src/main/java/de/xikolo/models/Item.java

```diff
@@ -112,6 +115,25 @@ public int getIconRes() {
         return icon;
     }
 
+    public String formatTimeEffort() {
+        int hours = timeEffort / 3600;
+        int minutes = (int) Math.ceil((timeEffort % 3600) / 60.0);
+
+        if (hours > 0) {
+            if (minutes > 0) {
+                return String.format(App.getInstance().getString(R.string.item_duration_hour_minute), hours, minutes);
+            } else {
+                return String.format(App.getInstance().getString(R.string.item_duration_hour), hours);
+            }
+        } else if (minutes > 0) {
+            return String.format(App.getInstance().getString(R.string.item_duration_minute), minutes);
+        } else {
+            return null;
+        }
+
+
```

Comment:
Don't add empty lines at the end of a method.

Reply:
solved